style.css

```diff
@@ -0,0 +1,728 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+}
+
+img,
+picture,
+video,
+canvas,
+svg {
+  display: block;
+  max-width: 100%;
+}
+
+input,
+button,
+textarea,
+select {
+  font: inherit;
+}
+
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  overflow-wrap: break-word;
+}
+
+#root,
+#__next {
+  isolation: isolate;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+body {
+  font-family: "Inter", sans-serif;
+  overflow-x: hidden;
+  line-height: 1.5;
+  -webkit-font-smoothing: antialiased;
+}
+
+.Container-1 {
+  position: absolute;
+  width: 375px;
+}
+
+p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.header-mobile {
+  height: 718px;
+  left: 0;
+  top: 0;
+  background: #fff;
+}
+
+header {
+  padding: 20px 20px;
+}
+
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.menu1 {
+  height: 48px;
+}
+
+.menu1 a {
+  display: flex;
+  width: 96px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #344563;
+}
+
+.navigation .logo {
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+}
+
+.menu1 ul {
+  display: none;
+}
+
+.main-part {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+  position: absolute;
+  height: 604px;
+  top: 92px;
+  background-image: url("media/header-shape-mobile-2x.svg");
+  background-repeat: no-repeat;
+}
+
+.main-part-title h1 {
+  padding: 0 24px 12px 24px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  display: flex;
+  align-items: center;
+}
+
+.main-part p {
+  padding: 0 24px 12px 24px;
+  font-size: 15px;
+  height: auto;
+  flex-direction: column;
+}
+
+.connect-let {
+  padding: 0 24px 12px 24px;
+  color: #7f8cff;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.s-m-i {
+  padding: 0 24px 12px 20px;
+  height: 20px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+  list-style-type: none;
+}
+
+.s-m-i img {
+  margin-right: 15px;
+}
+
+.jobs {
+  background: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+  left: 0;
+  top: 718px;
+}
```

Comment:
- [ ] Following the project requirements, you should use both Flexbox and Grid in order to lay out the elements on the page. In the previous milestones, you were requested to use "grid" for the "works" section. Kindly use Grid for this milestone as well.